common/README.md

```diff
@@ -2,8 +2,8 @@
 
 This crate contains several things right now:
 
-- Common structs that are used at API boundaries of higher-level crates
-  Putting the structs in `common` instead of in a larger crate allows to break
+- Common structs that are used at API boundaries of higher-level crates.
+  Putting the structs in `common` instead of in a larger crate allows us to break
```

Comment:
Since this sentence is being fixed, perhaps it should also get updated to be something like
```suggestion
  Putting the structs in `common` instead of in a larger crate allows us to break up
```

I don't think the intent was to "break" the dependencies so much as make them smaller